.gitignore

```diff
@@ -259,4 +263,477 @@ paket-files/
 
 # Python Tools for Visual Studio (PTVS)
 __pycache__/
-*.pyc
\ No newline at end of file
+*.pyc
+
+# Cake - Uncomment if you are using it
+# tools/
+
+### ReactNative ###
+# React Native Stack Base
+
+.expo
```

Comment:
We are not using react-native libraries here

Reply:
> 
> 
> We are not using react-native libraries here

will remove

Reply:
https://github.com/facebook/react/blob/master/.gitignore for react side, this one would be enough